app/views/docs/configuration.phtml

```diff
@@ -0,0 +1,65 @@
+<p>
+    Self-hosting allows you to customize Appwrite to meet your needs and preferences. Here's a list of things you will need to set up before developing with Appwrite. Before you start, back up your <code>.env</code> and <code>docker-compose.yml</code> files.
+</p>
+
+<h2><a href="#local-development" id="local-development">Local Development</a></h2>
+<p>Self-hosting Appwrite for development lets you quickly prototype on your local machine. Since localhost is only a loopback address that points back at your development machine, there are some caveats you should be aware of when working with localhost.</p>
+<p>Appwrite generates a self-signed certificate for iOS and Android development on localhost, as they don't allow unsecured HTTP by default. If you're developing Android or iOS apps on localhost, set <code>setSelfSigned(true)</code> while initializing your Appwrite client.</p>
+<p>If you're using an Android emulator to test your Flutter or Android apps, you can access localhost of your machine using the special alias IP <code>10.0.2.2</code>.</p>
+<p>If you're testing your Flutter, Android, or iOS app on a mobile device, you will not be able to access the localhost of your PC or Mac. When testing with a physical device, you may use a service like <a href="https://ngrok.com/" target="_blank" rel="noopener">ngrok</a> to proxy the Appwrite instance.</p>
+
+<h2><a href="#delivery" id="delivery">Setup Email and SMS Delivery</a></h2>
+<p>
+    Appwrite requires an SMTP service to deliver emails. Email delivery is required for features like password recovery, account verification, MagicURL authentication, and team invites. You can use a third-party SMTP service or host your own SMTP server.
+</p>
+<p><a href="/docs/email-delivery">Set up email delivery</a></p>
+
+<p>
+    Appwrite requires a third-party SMS delivery service to send SMS messages. SMS delivery is required for phone authentication.
```

Comment:
Can we rephrase it to sound more like a feature than a limitation ? 

`Appwrite supports multiple third-party SMS delivery providers .. ( flexibility, no-vendor locks etc.  )`

Reply:
I like this.